jscomp/test/test_obj_simple_ffi.js

```diff
@@ -2,12 +2,15 @@
 
 
 function v(displayName, _) {
-  return {
-          displayName: displayName ? displayName[0] : undefined,
-          test: 3,
-          config: 3,
-          hi: "ghos"
-        };
+  var $js = {
+    test: 3,
+    config: 3,
+    hi: "ghos"
+  };
+  if (displayName) {
+    $js.displayName = displayName;
```

Comment:
This isn't unboxing the parameter, I believe it should be `$js.displayName = displayName[0];`